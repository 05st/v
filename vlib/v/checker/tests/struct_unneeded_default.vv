struct Test {
	n int = 0
	s string = ''
}

fn main() {
}
