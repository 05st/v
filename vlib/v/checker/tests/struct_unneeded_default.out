vlib/v/checker/tests/struct_unneeded_default.vv:2:2: error: unnecessary default value of `0`: struct fields are zeroed by default
    1 | struct Test {
    2 |     n int = 0
      |     ~~~~~~
    3 |     s string = ''
    4 | }
